test-suite/Makefile

```diff
@@ -62,6 +62,10 @@ coqtopbyte := $(BIN)coqtop.byte -q
 coqc_interactive := $(coqc) -test-mode -async-proofs-cache force
 coqdep := $(BIN)coqdep
 
+# This is the convention for coq_makefile
+OPT=-$(BEST)
+export OPT
```

Comment:
Shouldn't coq_makefile handle this automatically? cc @coq/coq-makefile-maintainers

Reply:
I don't know the opinion of @coq/coq-makefile-maintainers but since the convention of using `OPT` in `coq_makefile` is well established, it ever we want to keep only one of `OPT` and `BEST`, I would lean towards keeping only `OPT`.